Documentation/getting_started/2_understanding_the_code.md

```diff
@@ -0,0 +1,90 @@
+
+This file will go over the code that is getting created when you start a blank project. For help on creating a project please look at [Creating a New Project](getting_started/1_creating_a_new_project.md)
+
+**Using Statements**
+```c#
```

Comment:
The two I've seen supported there are `c#` or `csharp`.  I don't think I've seen `cs` ever.